social-tw/social-tw-website
# fix: ignored yarn-error.log

## Summary

Added `yarn-error.log` to `.gitignore` and removed `packages/frontend/yarn-error.log`.

## Linked Issue

#72 

## Details

To prevent unnecessary tracking of yarn errors in the repository, the `yarn-error.log` has been ignored and a previously committed log has been removed.

## Impacted Areas

- `.gitignore`
- `packages/frontend/yarn-error.log`

## Tests

- No new tests were needed for this change.
- Existing tests should run without any issue.

## Possible Impacts

Other developers might need to pull the latest changes to see the effect of `.gitignore` and ensure they do not push any further `yarn-error.log`.

## Visual Materials

No UI effects for these changes.

## Verification Steps

1. Pull the latest changes.
2. Check that `yarn-error.log` is present in `.gitignore`.
3. Ensure `packages/frontend/yarn-error.log` is no longer in the repository.

## Todo

- [x] Ensure no other occurrences of `yarn-error.log` in the repository.
- [ ] Confirm with team members about the change.

## Checklist
